samples/dymaptic.GeoBlazor.Core.Sample.TokenRefresh/dymaptic.GeoBlazor.Core.Sample.TokenRefresh/Controllers/ApiController.cs

```diff
@@ -0,0 +1,75 @@
+using dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Controllers;
+
+/// <summary>
+/// API controller providing both authentication and configuration endpoints for the TokenRefresh sample.
+/// 
+/// Configuration Architecture:
+/// - Server stores sensitive credentials (ArcGISAppId, ArcGISClientSecret) in appsettings.json or user secrets
+/// - Client retrieves non-sensitive config (GeoBlazor license, ArcGIS API key, Portal URL, App ID) via /api/config endpoint
+/// - Client calls /api/auth/token endpoint to request tokens without exposing credentials
+/// 
+/// This separation ensures:
+/// - Client secrets remain secure on the server
+/// - WebAssembly client gets necessary configuration for GeoBlazor initialization
+/// - Token requests are authenticated server-side using client credentials flow
+/// </summary>
+[ApiController]
+[Route("api")]
+public class ApiController : ControllerBase
+{
+    private readonly ArcGisAuthService _authService;
+    private readonly IConfiguration _configuration;
+    
+    public ApiController(ArcGisAuthService authService, IConfiguration configuration)
+    {
+        _authService = authService;
+        _configuration = configuration;
+    }
+    
+    /// <summary>
+    /// Retrieves client configuration needed for WebAssembly application initialization.
+    /// Returns non-sensitive configuration values that the client needs to configure GeoBlazor.
+    /// </summary>
+    [HttpGet("config")]
+    public IActionResult GetClientConfig()
```

Comment:
I would really prefer these two endpoints be minimal APIs like in our other projects instead of a Controller.